web/src/js/components/Modal/Modal.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import * as modalAction from '../../ducks/ui/modal'
+import ModalList from './ModalList'
+
+class PureModal extends Component {
+
+    constructor(props, context) {
+        super(props, context)
+    }
+
+    render() {
+        const { activeModal, hideModal } = this.props
+        const ActiveModal = _.find(ModalList, m => m.name === activeModal )
```

Comment:
I'm afraid it looks weired if we add a fallback component in this way, because we have ` <div className="modal-backdrop fade in"></div>` below, which will make the background goes dark. 
How about keep this line and changing the retun like:
```javascript
return(
    activeModal && ActiveModal ? ():<div/>
)
```